.travis.yml

```diff
@@ -5,7 +5,9 @@ language: python
 cache:
   - directories:
     - $HOME/.cache/pip
-
```

Comment:
From https://github.com/falconry/falcon/blob/master/.travis.yml#L78-L79 : 
> NOTE(vytas): wheels sometimes fail to build on s390x due to cache being owned by another user.

So, perhaps these two new lines can be removed on the amd64 architecture.

Reply:
It works without:
* https://travis-ci.org/github/dalf/searx/builds/670155742/config
* https://github.com/dalf/searx/blob/return42-rfc-travis/.travis.yml

Reply:
Thanks a lot for the tested solution .. I applied 41f002b